sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/validator/PlatformKey.java

```diff
@@ -26,6 +26,8 @@
 @Retention(RUNTIME)
 @Constraint(validatedBy = KeyValidator.class)
 @Documented
+@Deprecated()
```

Comment:
Do we really want to deprecate the single key? I can imagine that you want to use exactly one key in many cases. Is it much work maintaining both annotations?

Reply:
no, it is ok for me. it's not extra work